parse/parse.go

```diff
@@ -9,6 +9,8 @@ import (
 	"strings"
 	"unicode"
 
+	"github.com/theothertomelliott/soy/errortypes"
```

Comment:
Robfig?

Reply:
Ah, perils of forking. Moved to the correct path and fixed all these.